templates/content/field--oe-document-reference--oe-documents--oe-document-group.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Field template of the document reference document group entity.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}
+  <div class="ps-4{{ not loop.last ? ' mb-3' }}">
```

Comment:
Let's drop the ps-4
![image](https://user-images.githubusercontent.com/8488617/178140500-b1b997af-39c6-49a7-826c-a811f76f334b.png)

Reply:
@brummbar Without the ps-4, there is no clear distinction where the group ends before single documents.
I don't like it either. Let's see what the UX team comes up with.

Reply:
For now we remove.